compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -1467,7 +1467,12 @@ fn print_native_static_libs(
     }
 
     match out {
-        OutFileName::Real(_) => out.overwrite(&lib_args.join(" "), sess),
+        OutFileName::Real(path) => {
+            out.overwrite(&lib_args.join(" "), sess);
```

Comment:
Can you move this into the if branch? Also I'm not sure what exact form `lib_args` has and whether it's straightforward to find the native artifacts in the file after reading the note on stdout, would it maybe be better to write this to the file: `"native-static-libs: {}", &lib_args.join(" ")`?

Reply:
ah nvm, I don't think this is necessary.